Sources/ProjectDescription/CompatibleXcodeVersions.swift

```diff
@@ -5,55 +5,76 @@ public enum CompatibleXcodeVersions: ExpressibleByArrayLiteral, ExpressibleByStr
     /// The project supports all Xcode versions.
     case all
 
+    case exact(Version)
+
+    case upToNextMajor(Version)
+
+    case upToNextMinor(Version)
+
     /// List of versions that are supported by the project.
-    case list([String])
+    case list([CompatibleXcodeVersions])
```

Comment:
Don't think it makes sense to change this right?

Wouldn't this allow:

```
compatabileXcodes: [.upToNextMajor("1.0.0"), .upToNextMinor("1.0.0")]
```

Reply:
Is there a problem with allowing that? It's likely useless, but not invalid.

The purpose of allowing this form is something like the above example:

```swift
compatibleXcodeVersions: [.upToNextMajor("13.2"), "12.5.1"]
```

Reply:
Its not a problem since it will resolve correctly and I don't imagine it'll be a common usage